libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -154,8 +154,10 @@ private static PolicyManager createPolicyManager() {
             serverModuleFileDatas,
             // Base ES directories
             FileData.ofPath(bootstrapArgs.pluginsDir(), READ),
+            FileData.ofPath(bootstrapArgs.modulesDir(), READ),
```

Comment:
Is this due to PluginsService and we just weren't checking it before?

Reply:
Yes, correct